api/controllers/emailSubscribe.js

```diff
@@ -1,5 +1,4 @@
-var auth = require('../helpers/auth');
-var _ = require('lodash');
+var { remove, indexOf, assignIn } = require('lodash');
```

Comment:
why was this done instead of _.remove, _.indexOf, _.assignIn, this doesn't seem to align with their docs.

Reply:
I was attempting to "tree shake" the dependencies of lodash to give us less code to ship. After we discussed this I actually looked into this further and this isn't the way to prune unneeded code with lodash. They recommend a different method here: https://lodash.com/per-method-packages